ci/build.sh

```diff
@@ -54,13 +54,17 @@ fi
 echo "version: $jdk_version"
 
 echo "Adding openjdk to bosh blobs"
-#jdk_blob_filename="jdk-${jdk_version}.tar.gz"
-jre_blob_filename="jre-${jdk_version}.tar.gz"
+jdk_blob_filename="jdk-${jdk_version}.tar.gz"
+jre_blob_filename="jre-${jre_version}.tar.gz"
```

Comment:
Do we need JRE blob anymore? It looks like we don't include it in the package anymore. Can it be removed from config/blobs.yml

Reply:
We don't need the JRE blob anymore as it's already included in the JDK blob. Feel free to remove it.